tasks/apt.yml

```diff
@@ -6,7 +6,7 @@
   notify: reload systemd
   with_items:
     - sudo
-    - python-software-properties
+    - software-properties-common
```

Comment:
@nemesisdesign  `python-software-properties` is deprecated so replaced it with the new `software-properties-common`